test/python/test_gravity_model.py

```diff
@@ -171,3 +171,39 @@ def test_polyhedral_evaluable_pickle(
     acceleration = np.array([result[1] for result in sol])
     np.testing.assert_array_almost_equal(potential, expected_potential)
     np.testing.assert_array_almost_equal(acceleration, expected_acceleration)
+
+
+def test_polyhedron_metric() -> None:
+    """Tests the metric conversion/ options
+    """
+    computation_point = np.array([1.0, 0.0, 1.0])
+    gravity_constant = 6.67430e-11
+    meter_polyhedron = Polyhedron(
+        polyhedral_source=(CUBE_VERTICES, CUBE_FACES),
+        density=1.0,
+        integrity_check=PolyhedronIntegrity.DISABLE,
+        metric_unit=MetricUnit.METER,
+    )
+    meter_evaluable = GravityEvaluable(polyhedron=meter_polyhedron)
+    kilometer_polyhedron = Polyhedron(
+        polyhedral_source=(CUBE_VERTICES, CUBE_FACES),
+        density=1.0,
+        integrity_check=PolyhedronIntegrity.DISABLE,
+        metric_unit=MetricUnit.KILOMETER,
+    )
+    kilometer_evaluable = GravityEvaluable(polyhedron=kilometer_polyhedron)
+    unitless_polyhedron = Polyhedron(
+        polyhedral_source=(CUBE_VERTICES, CUBE_FACES),
+        density=1.0,
+        integrity_check=PolyhedronIntegrity.DISABLE,
+        metric_unit=MetricUnit.UNITLESS,
+    )
+    unitless_evaluable = GravityEvaluable(polyhedron=unitless_polyhedron)
+
+    assert evaluate(meter_polyhedron, computation_point)[0] * 1e-9 == evaluate(kilometer_polyhedron, computation_point)[0]
```

Comment:
is it reasonable to use == here?

typical I would use np.isclose for FP comparisons

Reply:
Agreed